feat/time_directive - added #time directive (#37)
fsprojects/fsharp-cheatsheet: docs/fsharp-cheatsheet.md

### Single-case active patterns

> *Single-case active patterns* can be thought of as a simple way to convert data to a new form.

Reviewer: ```suggestion
*Single-case active patterns* can be thought of as a simple way to map data to a relevant form within a pattern match expression, parameter binding, or argument binding
```

### Partial active patterns

*Partial active patterns* share the syntax of parameterized patterns, but their active recognizers accept only one argument.

> The *Partial active pattern* should return an `Option<'T>`.

Reviewer: ```suggestion
A *Partial active pattern* must return an `Option<'T>`.
```

## Compiler Directives

### time

> Available in the FSI, is `#time` for a quick view into real time, CPU time, and garbage collection information.

Reviewer: ```suggestion
The `dotnet fsi` directive, `#time` switches on basic metrics covering real time, CPU time, and garbage collection information.
```

    #time
    System.Threading.Thread.Sleep (System.TimeSpan.FromSeconds 1)
    #time

    // Output:
    // --> Timing now on
    // Real: 00:00:01.001, CPU: 00:00:00.000, GC gen0: 0, gen1: 0, gen2: 0
    // val it: unit = ()
    // --> Timing now off

### load

Load another F# source file into FSI.

    #load "../lib/StringParsing.fs"

> ### Reference

Reviewer: ```suggestion
### Referencing packages or assemblies in a script
```
